### ISSUE TITLE
Fast intermediates do not display

### ISSUE THREAD

**Issue by psychedelicious** [invoke-ai/InvokeAI#1212]
@damian0815

Making an issue for this for future implementation. 

Branch https://github.com/damian0815/InvokeAI/tree/show-latent-steps

The issue is related to the failure of my server do do things async. A minimum of ~sleep(0.2) gives the server and UI enough chances to load the image. I'm not sure if it can really be fixed, but I believe the upcoming node backend does not have this issue.

**Comment by damian0815** [comment]
let's not stress on this. i think it will give InvokeAI an advantage in terms of the grokkability of what SD is doing (especially with img2img) but it's not at all critical for 2.1

**Comment by psychedelicious** [comment]
Agree, I think we will need to skip this entirely because the blocker is a systemic issue; new backend I believe will not have this issue. 

Actually, a very simple alternative solution is to just send the small intermediate image as base64, it's small enough to happily fit in a websocket signal. I'll investigate this when there is time for experimentation.

**Comment by damian0815** [comment]
we should chat more about this because after implementing cross-attention control i have ideas for like manipulating attention maps in real-time - like, pausing the generation process and allowing the user to paint over and otherwise manipulate the attention maps and/or the latents directly. this would absolutely be better done by pushing things through the socket rather than loading/saving images

**Comment by lstein** [comment]
I'm moving this to Discussions so that it doesn't get closed out inadvertently.
